python/src/com/jetbrains/python/console/PydevConsoleRunnerFactory.java

```diff
@@ -112,8 +112,9 @@ public PydevConsoleRunnerImpl createConsoleRunner(@NotNull Project project,
     putIPythonEnvFlag(project, envs);
 
     Consumer<String> rerunAction = title -> {
-      PydevConsoleRunnerImpl runner = createConsoleRunner(project, module);
-      runner.setConsoleTitle(title);
+      PydevConsoleRunner runner = createConsoleRunner(project, module);
+      if(runner instanceof PydevConsoleRunnerImpl)
+        ((PydevConsoleRunnerImpl)runner).setConsoleTitle(title);
```

Comment:
It's better to always have curly braces in Java.
Maybe we move setConsoleTitle to the PydevConsoleRunner method?